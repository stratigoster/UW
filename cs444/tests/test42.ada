PACKAGE foo IS
  procedure john (a,b:string := "hi"; c:integer := 5 ** 2 mod 3 + 4); -- default values work
  procedure doe (a,b : integer; c,d: string; a,b: float);
--                                           ^^^
-- duplicate variables a,b in parameter list
END foo;
