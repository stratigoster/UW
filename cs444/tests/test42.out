***ERROR***:3:57:	Erroneous repetition of parameters

------
Compilation error
------
